sql/src/main/antlr/OpenSearchSQLParser.g4

```diff
@@ -433,6 +433,7 @@ multiFieldRelevanceFunctionName
     : MULTI_MATCH
     | SIMPLE_QUERY_STRING
     | QUERY_STRING
+    | QUERYSTRING
```

Comment:
nit: put `QUERY_STRING` and `QUERYSTRING ` on the same line

Reply:
I don't see querystring in the documentation, nor do I see it used anywhere in the engine.  I think this is another function that we can skip and isn't required for backwards compatibility. 
@Yury-Fridlyand thoughts?